app/assets/javascripts/components/jquery_wrappers/TaggedInput.jsx

```diff
@@ -1,33 +1,30 @@
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
 
-class TaggedInput extends React.Component {
-  constructor(props) {
-    super(props);
-    this.saveInputRef = input => {
-      this.input = input;
-    };
-  }
+const TaggedInput = ({ availableLabels, disabled, onChange, input }) => {
+  const inputRef = useRef(null);
+
+  const tagitProperties = {
+    availableTags: availableLabels,
+    readOnly: disabled,
+  };
 
-  componentDidMount() {
-    this.loadTagit();
-  }
+  const loadTagit = () => {
+    if (inputRef.current) {
+      $(inputRef.current).tagit(tagitProperties).on('change', onChange);
+    }
+  };
 
-  loadTagit() {
-    $(this.input)
-      .tagit(this.tagitProperties())
-      .on('change', this.props.onChange);
-  }
+  useEffect(() => {
+    loadTagit();
```

Comment:
Instead of creating a separate function for loadTagit, we can implement it directly here, like was done with `popover`.